sdk/src/wallet/operations/balance.rs

```diff
@@ -338,13 +346,18 @@ impl<S: 'static + SecretManage> Wallet<S> {
 
         #[cfg(not(feature = "participation"))]
         {
-            balance.base_coin.available = balance.base_coin.total.saturating_sub(locked_amount);
+            balance.base_coin.available = balance
+                .base_coin
+                .total
+                .saturating_sub(delegation_implicit_accounts_amount)
+                .saturating_sub(locked_amount);
```

Comment:
Would it not make sense to consider it locked?

Reply:
I mean, adding amounts to locked_amount directly

Reply:
Sure, done